README.md

```diff
@@ -83,6 +83,8 @@ Currently, the following modules have been implemented:
 - [length](https://github.com/diogomatoschaves/pyturf/tree/master/turf/length)
 - [midpoint](https://github.com/diogomatoschaves/pyturf/tree/master/turf/midpoint)
 - [nearest-point](https://github.com/diogomatoschaves/pyturf/tree/master/turf/nearest_point)
+- [point-on-feature](https://github.com/diogomatoschaves/pyturf/tree/master/turf/point_on_feature)
+- [nearest-point](https://github.com/diogomatoschaves/pyturf/tree/master/turf/nearest_point)
```

Comment:
`nearest-point` is being repeated.

Reply:
fixed. this is an artefact of the rebase. My fault.